.github/workflows/tests.yml

```diff
@@ -377,7 +377,7 @@ jobs:
           echo "root:1000000:1000000000" | sudo tee /etc/subuid /etc/subgid
           cd test
           export PATH="$PATH:/snap/microovn/current/commands"
-          sudo --preserve-env=PATH,GOPATH,GOCOVERDIR,GITHUB_ACTIONS,GITHUB_STEP_SUMMARY,LXD_VERBOSE,LXD_BACKEND,LXD_CEPH_CLUSTER,LXD_CEPH_CEPHFS,LXD_CEPH_CEPHOBJECT_RADOSGW,LXD_OVN_NB_CONNECTION,LXD_OVN_NB_CLIENT_CRT_FILE,LXD_OVN_NB_CLIENT_KEY_FILE,LXD_OVN_NB_CA_CRT_FILE,LXD_OFFLINE,LXD_SKIP_TESTS,LXD_REQUIRED_TESTS,GOTRACEBACK, LXD_BACKEND=${{ matrix.backend }} ./main.sh ${{ matrix.suite }}
+          sudo --preserve-env=PATH,GOPATH,GOCOVERDIR,GITHUB_ACTIONS,GITHUB_STEP_SUMMARY,LXD_VERBOSE,LXD_BACKEND,LXD_CEPH_CLUSTER,LXD_CEPH_CEPHFS,LXD_CEPH_CEPHOBJECT_RADOSGW,LXD_OVN_NB_CONNECTION,LXD_OVN_NB_CLIENT_CRT_FILE,LXD_OVN_NB_CLIENT_KEY_FILE,LXD_OVN_NB_CA_CRT_FILE,LXD_OFFLINE,LXD_SKIP_TESTS,LXD_REQUIRED_TESTS,GOTRACEBACK,LXD_CONCURRENT LXD_BACKEND=${{ matrix.backend }} ./main.sh ${{ matrix.suite }}
```

Comment:
I'm ambivalent with this `LXD_CONCURRENT` variable. Do we actually need a variable that catches us by surprise by causing some tests to not run?

Reply:
What does it do?

Reply:
The `LXD_CONCURRENT` flag is used to deactivate the `test_concurrent` and `test_concurrent_exec` tests.
Those 2 tests have an early `return` if the flag is not set which is what I don't like as it makes it easy to miss that those tests do nothing.

They take between 15s (`dir`) and 2m (`ceph`).

Reply:
> ok lets remove the env var

Nice!

FYI, I also don't like `LXD_SKIP_TESTS` nor `LXD_REQUIRED_TESTS`. IMHO all the tests should be run and required all the time. If we ever want to skip or disable some it should be done in a more explicit way.

Reply:
Hmm, I had not fully thought it through. Maybe we should keep `LXD_SKIP_TESTS` if we want to temporarily skip a test case. I'll have a think and come up with a PR for this.

I guess my main grip is with `LXD_REQUIRED_TESTS`. Aren't all the tests required, unless skipped?